.circleci/src/jobs/check_web.yml

```diff
@@ -1,3 +1,8 @@
+parameters:
```

Comment:
Why do we need a build config name here? This is executed independently of a build config name just like e.g. check_native

Reply:
I think I already removed it